src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.jsx

```diff
@@ -0,0 +1,9 @@
+import './PlanetName.css';
+
+const PlanetName = ({name}) => {
+    return (
+        <h1 class="planetName">{name}</h1>
```

Comment:
Component class name should be PascalCase

Reply:
Same for [PR #4](https://github.com/jfiora/the-planets/pull/4)

Reply:
done